code/game/gamemodes/miniantags/abduction/abduction_surgery.dm

```diff
@@ -49,6 +49,9 @@
 		to_chat(user, "<span class='warning'>You don't find anything in [target]'s [target_zone]!</span>")
 		return 1
 
+/datum/surgery_step/internal/extract_organ/fail_step(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
+	return end_step(user, target, target_zone, tool, surgery)
```

Comment:
I don't really like this. You can intentionally fail just to make the step shorter.

Reply:
Okay, I'll add a failure message instead